src/openforms/submissions/tests/test_admin.py

```diff
@@ -126,7 +129,7 @@ def test_export_xlsx_successfully_exports_xlsx_file(self):
 
     def test_exporting_multiple_forms_fails(self):
         step = FormStepFactory.create()
-        SubmissionFactory.create(form=step.form)
+        SubmissionFactory.create(form=step.form, completed_on=datetime.now())
```

Comment:
should use `django.utils.timezone`

```suggestion
        SubmissionFactory.create(form=step.form, completed_on=timezone.now())
```

Reply:
how did this get past this warning filter in the dev settings:

```py
# THOU SHALT NOT USE NAIVE DATETIMES
warnings.filterwarnings(
    "error",
    r"DateTimeField .* received a naive datetime",
    RuntimeWarning,
    r"django\.db\.models\.fields",
)
```